GeneratorInterface/GenFilters/interface/MCParticlePairFilter.h

```diff
@@ -38,6 +38,9 @@
 namespace edm {
   class HepMCProduct;
 }
+namespace HepMC {
```

Comment:
i just did. don't really understand it. will merge and we see in the future how things progress as compilers change